### PR TITLE
Fix/ask ai concepts

### PR DESCRIPTION
**Pull request by RybakovaIE**

### CODE REVIEW

File: sc-models/knowledge-base/concepts/relations.scs [hunk 1]
```diff
@@ -1,3 +1,9 @@
 nrel_ostis_system_decomposition
 <- sc_node_norole_relation;
 => nrel_main_idtf: [декомпозиция ostis-системы*] (* <- lang_ru;; *);;
+
+nrel_definition
+<- sc_node_norole_relation;
+=> nrel_main_idtf: 
+    [definition*] (* <- lang_en;; *);
+    [определение*] (* <- lang_ru;; *);;
```
Comment (MksmOrlov) [inline sc-models/knowledge-base/concepts/relations.scs:9]: Add newline. Check all files in your PR please

File: platform-dependent-components/problem-solver/cxx/message-reply-module/agent/MessageReplyAgent.cpp
Comment (MksmOrlov) [inline platform-dependent-components/problem-solver/cxx/message-reply-module/agent/MessageReplyAgent.cpp:1]: You already fixed it in https://github.com/ostis-ai/ostis-metasystem/pull/58/files

File: sc-models/problem-solver/message_reply/message_processing_program.scs
Comment (MksmOrlov) [inline sc-models/problem-solver/message_reply/message_processing_program.scs:1]: You already fixed it in https://github.com/ostis-ai/ostis-metasystem/pull/58/files

**Review by MksmOrlov — CHANGES_REQUESTED** [review]

**Comment by RybakovaIE** [comment]
https://github.com/ostis-ai/ostis-metasystem/pull/58

**Pull request closed without merging**